R/anovarepeatedmeasures.R

```diff
@@ -354,6 +354,11 @@ AnovaRepeatedMeasures <- function(jaspResults, dataset = NULL, options) {
   # Make sure that order of anova result corresponds to order of specified model terms
   mappedRownamesCases <- .mapAnovaTermsToTerms(rownames(sortedModel), unlist(modelDef$terms.base64))
 
+  # when interactions effects are excluded from the model terms they are still computed by afex
+  # their names are mapped to zero by .mapAnovaTermsToTerms so we first exclude them
+  idxValid <- which(mappedRownamesCases != 0L)
+  sortedModel <- sortedModel[idxValid, ]
```

Comment:
I don't need to use `which`, but the suggestion won't work because at line 376 `idxValid` is reused. Here also the [diff without whitespace changes](https://github.com/jasp-stats/jaspAnova/pull/64/files?diff=split&w=1).